### DISCUSSION TITLE
Feature Request: View day location breakdown

### DISCUSSION THREAD

**Discussion by CountParadox** [Freika/dawarich#553] [Ideas]
In google maps timeline; My most used feature is being able to go back over the last few days; and see what time I was at a certain place, I cannot find an easy way to do this here; I think it would be beneficial to have a day by day breakdown showing what time movement stopped at different locations. 
For example; this would be useful for completing timesheets for a job, or for proving that you arrived on time at a location.

**Comment by CountParadox** [comment]
I can supply screenshots from my google timeline if requested!

**Reply by Freika** [reply]
That would be useful, thanks.

**Comment by CountParadox** [comment]
![Screenshot_20241221_182843_Maps.png](https://github.com/user-attachments/assets/1a88d212-a928-49ae-b7ec-12e46d7610ef)

![Screenshot_20241221_182831_Maps.png](https://github.com/user-attachments/assets/7a2661ce-b149-4d3a-8427-03be96fedfaf)

Here's some screenshots, note you can see times I was at places.
